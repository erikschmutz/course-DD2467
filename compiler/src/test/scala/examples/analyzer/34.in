[|1; 2|]

(*====================

- :int[||]
====================*)