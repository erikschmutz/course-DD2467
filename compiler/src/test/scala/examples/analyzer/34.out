AnalyzerResult(ArrayList(Integer()),Enviroment(List()))